test/models/ping_thing_test.rb

```diff
@@ -0,0 +1,64 @@
+require "test_helper"
+
+class PingThingTest < ActiveSupport::TestCase
+  setup do
+    @user = create(:user)
+  end
+
+  test "create with correct data should succeed" do
+    assert PingThing.create(
+      amount: "1",
```

Comment:
creata hash `params` or sth like that in the setup method with params for PingThing (you can initialize with nils) and overwrite in particular tests as needed.